lib/coprl/presenters/plugins/theme/selector.rb

```diff
@@ -0,0 +1,23 @@
+module Coprl
+  module Presenters
+    module Plugins
+      module Theme
+        class Selector
+          attr_reader :context
+
+          include AAA::Helpers::AuthenticateAuthorize
+          include AppHelpers::ThemeHelper
```

Comment:
are these dependencies from the core app? we can't really rely on these here – pulling this plugin into a different project would fail.

Reply:
Appreciate the suggestion, fixed